pkg/proxy/iptables/proxier_test.go

```diff
@@ -2594,6 +2638,7 @@ func TestEndpointSliceE2E(t *testing.T) {
 :KUBE-SERVICES - [0:0]
 :KUBE-EXTERNAL-SERVICES - [0:0]
 :KUBE-FORWARD - [0:0]
+:KUBE-NODEPORTS - [0:0]
```

Comment:
can we add a test like `TestEndpointSliceE2E` but with a Service with health check node ports? Will help compare the whole rule set

Reply:
Good call, will do that.